roles/ceph-container-engine/tasks/pre_requisites/prerequisites.yml

```diff
@@ -4,6 +4,7 @@
   with_first_found:
     - "{{ ansible_facts['distribution'] }}-{{ ansible_facts['distribution_major_version'] }}.yml"
     - "{{ ansible_facts['os_family'] }}.yml"
+  when: container_package_name is undefined or container_service_name is undefined
```

Comment:
```suggestion
  when: container_package_name is undefined and container_service_name is undefined
```

Reply:
true. maybe adding a check in the ceph-validate role makes sense?

Reply:
@guits Added. PTAL